dev/tools/controllerbuilder/generate.sh

```diff
@@ -220,5 +220,15 @@ go run . generate-mapper \
      --service google.apps.cloudidentity.groups.v1beta1 \
      --api-version cloudidentity.cnrm.cloud.google.com/v1beta1
 
+# CloudIdentity : Membership
+go run . generate-types \
```

Comment:
nit: having a separate `generate-types` command for a resource within the same API only works if there is no existing `types.generate.go` file. The intended usage is to add the resource to the existing command, for example:  

```
go run . generate-types \
     --service google.apps.cloudidentity.groups.v1beta1 \
     --resource CloudIdentityGroup:Group \
     --resource CloudIdentityMembership:Membership \
     --api-version "cloudidentity.cnrm.cloud.google.com/v1beta1"
```

Also @gemmahou updated the "service" recently in #3676, so the command we are adding here no longer works with HEAD.